pennylane/ops/qubit/matrix_ops.py

```diff
@@ -211,25 +226,84 @@ def __init__(
         # to the left and right with the correct amount of identity blocks.
 
         self._padding_left = control_int * len(U)
-        self._padding_right = 2 ** len(wires) - len(U) - self._padding_left
-        self._CU = None
+        self._padding_right = 2 ** len(total_wires) - len(U) - self._padding_left
+        self._cu = None
 
-        super().__init__(*params, wires=wires, do_queue=do_queue)
+        super().__init__(*params, wires=total_wires, do_queue=do_queue)
 
     @property
     def num_params(self):
         return 1
 
-    def _matrix(self, *params):
-        if self._CU is None:
+    @staticmethod
+    def compute_matrix(U, control_wires, wires, control_values=None):
+        """Canonical matrix representation of the ControlledQubitUnitary operator.
+
+        Args:
+            U (tensor_like): unitary matrix
+            control_wires (Iterable): the control wire(s)
+            wires (Iterable): the wire(s) the unitary acts on
+            control_values (str): a string of bits representing the state of the control
+                qubits to control on (default is the all 1s state)
+
+        Returns:
+            tensor_like: canonical matrix
+
+        **Example**
+
+        >>> U = np.array([[ 0.94877869,  0.31594146], [-0.31594146,  0.94877869]])
+        >>> qml.ControlledQubitUnitary.compute_matrix(U, control_wires=[1], wires=[0], control_values="1")
```

Comment:
I find it funny that this op has separate `control_wires`, whereas other ops don't; the control wire is inferred from the order.

Reply:
Yes that's true! But I guess that's how the CNOT representation is canonically done (control on first), while this gate is something we invent and where the control position is variable...